src/components/AssetsComponent.jsx

```diff
@@ -69,53 +66,32 @@ export default class AssetsComponent extends Component {
     const showPaginator = totalPages > 0;
     const currentAssets = `page_${this.props.activePage}`;
     const { assets } = this.state;
+
     return (
       <NavBarComponent title="Assets">
         <div className="assets-list">
           <div id="page-heading-section">
-            <Header as="h1" id="page-headings" floated="left" content="Assets List" />
+            <Header as="h1" id="page-headings" floated="left" content="Assets" />
             <Divider id="assets-divider" />
-            <FilterButton
-              activePage={this.props.activePage}
-              limit={this.state.limit}
-              selected={this.props.selected}
-              filterAction={this.props.getAssetsAction}
-              disabled={this.props.isLoading}
-            >
-              <React.Fragment>
-                <FilterComponent
-                  index={0}
-                  option={this.props.filterData[0]}
-                  selected={this.props.selected}
-                  filterSelection={this.props.filterSelection}
-                />
-
-                <FilterComponent
-                  index={1}
-                  option={this.props.filterData[1]}
-                  selected={this.props.selected}
-                  filterSelection={this.props.filterSelection}
-                />
-              </React.Fragment>
-            </FilterButton>
           </div>
-          <AssetsTableContent
+          <AssetsTabComponent
             activePage={this.props.activePage}
-            assets={this.props.assetsList[currentAssets] || assets}
+            limit={this.state.limit}

```

Comment:
If most of these props just gets passed from here to  `AssetsTabComponent`, then I recommend creating a different container for `AssetsTab...`, that will retrieve the needed props from the store. First create the `AssetsFilterContainer` as per my previous component and see how many remaining props just gets passed along. Having a component expect props that it just passes to its children without using them should be a red flag that you need to decompose the component further.

Reply:
Done.